assets/js/modules/analytics/datastore/accounts.js

```diff
@@ -123,6 +137,20 @@ export const reducer = ( state, { type, payload } ) => {
 			};
 		}
 
+		case RESET_ACCOUNTS: {
+			return {
+				...state,
+				accounts: undefined,
+				settings: {
+					...state.settings,
+					accountID: undefined,
+					propertyID: undefined,
+					internalWebPropertyID: undefined,
+					profileID: undefined,
```

Comment:
I think resetting the settings here is a side effect that maybe shouldn't happen here - this should only reset accounts.